cap_anndata/cap_anndata.py

```diff
@@ -290,3 +290,20 @@ def obs_keys(self) -> List[str]:
 
     def var_keys(self) -> List[str]:
         return self.var.column_order.tolist()
+
+    def create_repr(self) -> str:
+        indent = " " * 4
+        s = f"CapAnnData object"
+        s += f"\n{indent}X shape: {self.shape}"
+        s += f"\n{indent}Has raw X: {self.raw is not None}"
+        for field in ["obs", "obsm", "var", "uns", "layers"]:
+            if field in self._file:
+                keys = list(self._file[field].keys())
+                s += f"\n{indent}{field}: {keys}"
```

Comment:
If you have time it will also nice to mark in-memory columns somehow. Maybe smh like:
```
obs: ['assay'*, 'cell_barcode', 'cell_label'...
...
* - in memory
```

Reply:
added